CMakeLists.txt

```diff
@@ -85,7 +85,10 @@ add_executable(deepseg
   app/background.cc
   app/calcmask.cc
   app/utils.cc
+  app/blend.cc
```

Comment:
What about keeping this list sorted by name?
(Fine with the exception for the main function).

Reply:
of course..